tox.ini

```diff
@@ -10,3 +10,4 @@ deps =
 commands =
   pytest --ignore setup.py {posargs}
   pytest urlextract {posargs: --doctest-modules}
+  black urlextract --skip-string-normalization
```

Comment:
Let's move this to `[testenv:black]`

Reply:
And also lets move  `--skip-string-normalization` to `pyproject.toml` file to `[tool.black]`. So we can set more things for Black in the future.

Reply:
Please check the updates @lipoja

Reply:
great! Thanks, could you also add `tests` folder as well?

Reply:
@lipoja OK, added!